cms/djangoapps/contentstore/views/item.py

```diff
@@ -654,6 +656,15 @@ def _save_xblock(user, xblock, data=None, children_strings=None, metadata=None,
         if grader_type is not None:
             result.update(CourseGradingModel.update_section_grader_type(xblock, grader_type, user))
 
+        # If visible_to_staff_only has changed, send a signal to regrade the course
+        if _block_visibility_changed(xblock, metadata, old_metadata):
+            COURSE_BLOCK_VISIBILTY_CHANGED.send(
+                user_id=user.id,
+                sender="_save_xblock",
+                course_key=xblock.location.course_key,
```

Comment:
nit: these args aren't ordered alphabetically or grouped by function (in-signal vs structural).

e.g.: { course_key, user_id, event_transaction_id, sender } or { course_key, event_transaction_id, sender, user_id }

Reply:
tbh i just ordered them by line length